CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## lifebit-ai/cloudos-py: changelog
 
+### 0.1.1 - 2022-07-12
+- Adds WDL pipeline support, iteration 1: cromwell server managing.
+Now, a new command `cloudos cromwell` is available, with the following
+subcommands:
+    * status
+    * restart
```

Comment:
@dapineyro let's mirror the UI and name it `start` 
<img width="398" alt="image" src="https://user-images.githubusercontent.com/38183826/179207991-f62a06b6-2a3d-4286-acc2-52e924672906.png">